hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerRatisServer.java

```diff
@@ -186,7 +189,7 @@ public static OzoneManagerRatisServer newOMRatisServer(
       raftPeers.add(raftPeer);
     }
 
-    return new OzoneManagerRatisServer(ozoneConf, om, omServiceId,
+    return new OzoneManagerRatisServer(ozoneConf, omProtocol, omServiceId,
```

Comment:
I think here it should be om? or we should change the method parameter name to omProtocol.
Currently code is not getting compiled.